src/platform/site-wide/announcements/selectors.js

```diff
@@ -24,7 +58,8 @@ export const selectAnnouncement = (
   if (announcements.isInitialized) {
     announcement = config.announcements
       .filter(a => !a.disabled)
-      .filter(isExpiredAnnouncement)
+      .filter(isStarted)
+      .filter(isNotExpired)
```

Comment:
A little confusing to have `Not` in the function name but I like the way this reads

Reply:
Indeed... I can't say `isExpired` though since that's the exact opposite of what that function does 🤔lmk if you think of a better var name!